vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -34,6 +47,177 @@
     fused_moe_pallas = None  # type: ignore
 logger = init_logger(__name__)
 
+MOE_DP_CHUNK_SIZE = 256
```

Comment:
Would be good to quickly document why this value

Reply:
I think this is just the value I picked arbitrarily when I first implemented the activation chunking